### DISCUSSION TITLE
Month in Revolt - August 2024 - Revolt

### DISCUSSION THREAD

**Discussion by giscus** [stoatchat/discussions#768] [Announcements]
# Month in Revolt - August 2024 - Revolt

The monthly blog update for July–August period.

https://revolt.chat/posts/month-in-revolt-august-2024

**Comment by Andre601** [comment]
Look mom. I'm in a blog post!

**Comment by amycatgirl** [comment]
They are finally adding proper documentation to Autumn and January! I hope the migration ends up well 🤞

**Comment by MMachado05** [comment]
Super exciting stuff!

However, I should mention that RSS is still broken. 😢 I had to view this here on my browser since all the markdown header things don't render correctly on my RSS reader.

**Comment by zmef** [comment]
Hi! revolt.chat is such a great piece of software! Super impressed with the work y'all are doing! Thanks so much! Check out the Software Freedom Day Capture the Flag support server here: [https://rvlt.gg/WTPkcdnD](https://rvlt.gg/WTPkcdnD)

**Comment by Daniel-A-Gutierrez** [comment]
Is it possible to restrict the creation of new accounts on a revolt server? I want to self host for my friends but I don't want any rando on the internet to be able to create an account on my instance.

**Reply by amycatgirl** [reply]
It is possible by setting your instance as "Invite Only"

See [here for intructions.](https://github.com/revoltchat/self-hosted?tab=readme-ov-file#making-your-instance-invite-only)
